training_toolbox/lpr/toolbox/spatial_transformer_layer.py

```diff
@@ -0,0 +1,203 @@
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Looks like it is the same file: `https://github.com/tensorflow/models/blob/master/research/transformer/spatial_transformer.py`. We use `models` repository as a submodule. Can we use the original file?

Reply:
Fixed.